docs/modules/ROOT/pages/use-cases.adoc

```diff
@@ -58,8 +58,8 @@ We will use the following template to describe the relevant use cases:
 [[sec:uc1]]
 [[sec:addressed-use-cases]]
 === Addressed use cases
-image::images/image-2025-02-27-14-08-05-543.png[]
 
+image::image-2025-02-27-14-08-05-543.png[]
```

Comment:
If we would have meaningful image file names, as mentioned in a previous comment, we would not need to update the documents that refer to the images, just because we update an image.